src/rust/sysmon-subgraph-generator/src/main.rs

```diff
@@ -505,18 +547,6 @@ where
     metrics: SysmonSubgraphGeneratorMetrics,
 }
 
-impl<C> Clone for SysmonSubgraphGenerator<C>
-where
-    C: Cache + Clone + Send + Sync + 'static,
-{
-    fn clone(&self) -> Self {
-        Self {
-            cache: self.cache.clone(),
-            metrics: self.metrics.clone(),
-        }
-    }
-}
-
```

Comment:
Not sure why `Clone` was explicitly implemented but `#[derive(Clone)]` worked fine. As long as all the members implement `Clone`, the parent should be able to implement `Clone` without explicit intervention.